chartpress.py

```diff
@@ -458,6 +497,9 @@ def build_images(
     force_build=False,
     skip_build=False,
     long=False,
+    *,
+    builder=BUILDERS[0],
```

Comment:
You made an if statement referencing "docker-build" the string öiteral above, but here you reference the constant. A bit inconsistent, perhaps go string literal here also for readability?